include/proxy/hdrs/MIME.h

```diff
@@ -1652,7 +1636,7 @@ MIMEHdr::value_append(const char *name, int name_length, const char *value, int
 inline time_t
 MIMEHdr::get_age() const
 {
-  int64_t age = value_get_int64(MIME_FIELD_AGE, MIME_LEN_AGE);
+  int64_t age = value_get_int64(std::string_view{MIME_FIELD_AGE, static_cast<std::string_view::size_type>(MIME_LEN_AGE)});
```

Comment:
This is another example where also having the other override is cleaner.

Reply:
I thought we could add string_view version of mime field constants once this pull request is merged.
After that, we could deprecate `const char * MIME_FIELD_XXX` and `int MIME_LEN_XXX`.

Reply:
I've made the changes I said above at https://github.com/apache/trafficserver/compare/038258a62b5a3663ea1aa68e443866da02552419...hnakamur:trafficserver:take_string_view_in_mime_methods2

Reply:
I added `c_str_view` which is similar to `std::string_view` but asserts the string is terminate with `\0`.
It is needed in src/api/InkAPIInternal.cc like:
```
TS_MIME_FIELD_ACCEPT                    = MIME_FIELD_ACCEPT.c_str();
```

The type of `TS_MIME_FIELD_ACCEPT` is `const char *` and it expected the string is terminated with `\0`.
The type of `MIME_FIELD_ACCEPT` was changed to `c_str_view` which expresses so.